Engine/main/update.cpp

```diff
@@ -291,7 +293,9 @@ void update_speech_and_messages()
 void update_sierra_speech()
 {
   // we need to know if there is/was voice-over
-  const bool is_voice = channel_has_clip(SCHAN_SPEECH);
+  AudioChannelsLock lock;
+  auto *ch = lock.GetChannel(SCHAN_SPEECH);
+  const bool is_voice = ch != nullptr;
```

Comment:
Similarly here, you could grab the speech pos here and free the lock.

Reply:
Right, so I thought about that, and my next pr with nonblocking voice-over adds different way to test for voice mode, so some of these checks become unnecessary and the place when you need to access channel also changes, so I think I'll leave this as it is for now (some code is going to be removed anyway)